server.js

```diff
@@ -146,7 +146,7 @@ app.get('/api/info', function(req, res) {
 app.get('/api/auth/:path', function(req, res) {
   auth.checkUrl(req, function(err, output) {
     if (err) {
-      return serverUtils.serverError(err, req, res);
+      return serverUtils.error(err, req, res);
```

Comment:
I think it would be better to fix the `serverError` implementation (or remove it altogether). It's quite confusing right now...